_data/tools/puppet-summary.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "puppet-summary",
+    "display_name": "Puppet Summary",
+    "description": "Another web interface for Puppet reports",
+    "url": "https://github.com/skx/puppet-summary"
+  }
```

Comment:
same here

Reply:
Done, had to force push to verify previous commits